PicPi/PictureFrameImpl.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  PictureFrameImpl.swift
+//  PicPi
+//
+//  Created by Erick Sanchez on 12/23/20.
+//
+
+import Promises
+
+func injectPictureFrame(ip: String, host: URL) -> PictureFrame {
+  PictureFrameImpl(frameAPI: injectFrameAPI(host: host), ip: ip)
+}
+
+private class PictureFrameImpl: PictureFrame {
+  var id: String { ip }
+  var name: String { ip }
```

Comment:
so id is the IP , and name is IP too , until we set the name or their should be a default name?

Reply:
for now the name is the ip but this will change to be user defined, haven't figured that out since idk how the wifi discovery works

Reply:
Spoke offline, planned to use MAC addresses for PictureFrame:id (more on this here: https://trello.com/c/tZFYawVA)